translations/mod-fqm-manager/en.json

```diff
@@ -335,5 +335,14 @@
   "entityType.drv_organization_details.expected_activation_interval": "Organization expected activation interval",
   "entityType.drv_organization_details.vendor_currencies": "Organization vendor currencies",
   "entityType.drv_organization_details.tax_id": "Organization tax ID",
-  "entityType.drv_organization_details.tax_percentage": "Organization tax percentage"
+  "entityType.drv_organization_details.tax_percentage": "Organization tax percentage",
+  "entityType.drv_organization_details.agreements": "Organization agreements",
+  "entityType.drv_organization_details.agreements.name": "Vendor terms name",
+  "entityType.drv_organization_details.agreements.name._qualified": "Organization vendor terms name",
+  "entityType.drv_organization_details.agreements.discount": "Discount percentage",
+  "entityType.drv_organization_details.agreements.discount._qualified": "Organization discount percentage",
+  "entityType.drv_organization_details.agreements.referenceUrl": "Reference URL",
+  "entityType.drv_organization_details.agreements.referenceUrl._qualified": "Organization reference URL",
+  "entityType.drv_organization_details.agreements.notes": "Notes",
+  "entityType.drv_organization_details.agreements.notes._qualified": "Organization notes"
```

Comment:
Would it be organization? Or something like "Vendor terms notes" instead? What describes this column on its own

Reply:
its inside agreements array

Reply:
then maybe "Agreements notes"?